book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Hello, 👏 it's truly commendable that you've included Getters for the @rentals instance! 🏡 This is great because it enables you to effortlessly access the current state of the rentals. However, keep in mind that you might also need to modify or update this state. To make this process smooth and maintain data integrity, it's a wise practice to implement Setters for the @rentals variable as well. 🛠️
- [ ] Setters act as your control center for making changes to the state of @rentals. 🔄 They allow you to update, add, or remove rental information as needed, ensuring that your data stays accurate and your rental operations run seamlessly. So, combining Getters and Setters creates a robust toolkit for effectively managing your rental system! 📊🔧